lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -42,7 +53,7 @@ def call
           CLI::UI.raw { print(ANSI.hide_cursor) }
           while @answer.nil?
             render_options
-            wait_for_actionable_user_input
+            wait_for_redraw
```

Comment:
I agree that `wait_for_actionable_user_input` no longer makes sense as a name, but `wait_for_redraw` looses something about reacting to user input.  Maybe `process_input_until_redraw_required`?  Something else along those lines?

Reply:
long_name_is_long, but sure.